docroot/modules/humsci/hs_bugherd/hs_bugherd.module

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file
+ * hs_bugherd.module
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\encrypt\Entity\EncryptionProfile;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function hs_bugherd_form_jira_rest_config_form_alter(&$form, FormStateInterface $form_state) {
+  if (\Drupal::moduleHandler()->moduleExists('encrypt')) {
```

Comment:
Instead of wrapping the entire contents of the function in this if statement it is usually easier for a developer to read and/or refactor if you checked if not exists and returned.